mobile/lib/features/wallet/create_invoice_screen.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:developer';
```

Comment:
One downside is that this is much longer than `create_invoice_screen`. `create_payment_request` also could maybe be confused with bitcoin, though this is arguably the same for invoices - personally though I don't think of bitcoin in terms of invoices, only lightning. Happy to change if you think it's best, though!

Reply:
It's just a nit 😅 I don't think its confusing as the term `payment_request` is also very much linked to lightning. The reason I suggested that change though, is because of the reference to the UI (see screenshot). There we are referencing to it as Payment Request and not Invoice. I would be fine either way, but having it consistent in code and UI could be helpful. (but again its just a nit - no need for action)